jenkins-x-integration.yml

```diff
@@ -49,14 +49,14 @@ pipelineConfig:
               - name: build
                 #image: docker.io/golang:1.11.5
                 # needs helm in the image for install_gitops_integration_test.go
-                image: gcr.io/jenkinsxio/builder-go-maven
+                image: gcr.io/jenkinsxio/builder-go-maven:0.0.0-SNAPSHOT-PR-1250-2
```

Comment:
I guess that's to test your builder changes, right? Any issues upgrading the builders?

Reply:
Yup. This won't build without Go 1.13, so I had to cheat.